libcxx/utils/synchronize_csv_status_files.py

```diff
@@ -284,7 +284,7 @@ def sync_csv(rows: List[Tuple], from_github: List[PaperInfo]) -> List[Tuple]:
             results.append(gh.for_printing())
             continue
         elif paper.status != gh.status:
-            print(f"We found a CSV row and a Github issue with different statuses:\nrow: {row}\Github issue: {gh}")
+            print(rf"We found a CSV row and a Github issue with different statuses:\nrow: {row}\Github issue: {gh}")
```

Comment:
This is not a regex -- this change looks wrong.

Reply:
See https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences. The `\n` and `\G` are interpreted as an escape sequence. Hence necessitates the change.

Reply:
I believe @negril is right.

Note that https://docs.python.org/3/whatsnew/3.12.html#other-language-changes doesn't say it's exclusive to regex:
> A backslash-character pair that is not a valid escape sequence now generates a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning), instead of [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning). For example, re.compile("\d+\.\d+") now emits a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) ("\d" is an invalid escape sequence, use raw strings for regular expression: re.compile(r"\d+\.\d+")). In a future Python version, [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError) will eventually be raised, instead of [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning). (Contributed by Victor Stinner in [gh-98401](https://github.com/python/cpython/issues/98401).)

Reply:
This doesn't look intended to me though. The `\G` should probably have been a `\nG`.

Reply:
That can be fix if you want me to?

Reply:
Ah, yeah that's the issue. The `\G` is unintended. I'll fix this right now.